From: Ondrej Jirman <megi@xff.cz>
Date: Tue, 21 Jun 2022 23:04:52 +0200
Subject: arm64: dts: rockchip: rk3399-pinephone-pro: Cleanup regulator sleep
 states

Forcing SD card I/O voltage to 3.3V in suspend is probably not a great
idea. A lot of properties were superfluous, etc.

Signed-off-by: Ondrej Jirman <megi@xff.cz>
---
 arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
index b8be1d1..ea2e16d 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-pinephone-pro.dts
@@ -396,10 +396,6 @@ vcc5v0_typec: vcc5v0-typec-regulator {
 		regulator-max-microvolt = <5000000>;
 		regulator-name = "vcc5v0_typec";
 		vin-supply = <&boost_otg>;
-
-		regulator-state-mem {
-			regulator-on-in-suspend;
-		};
 	};
 
 	vcc3v3_sys: vcc3v3-sys-regulator {
@@ -728,7 +724,6 @@ boost_otg: DCDC_BOOST {
 				regulator-max-microvolt = <5000000>;
 				regulator-state-mem {
 					regulator-on-in-suspend;
-					regulator-suspend-microvolt = <5000000>;
 				};
 			};
 
